src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/create-edit-or-find-alert-policy.mdx

```diff
@@ -136,3 +136,135 @@ The policies index lists policies in alphabetical order. To view or search for e
 <Callout variant="tip">
   To view [policy and condition information for a specific entity](/docs/alerts/new-relic-alerts-beta/configuring-alert-policies/view-policy-conditions-new-relic-products), select the **Settings > Alert conditions** page from the entity's product UI.
 </Callout>
```

Comment:
Not part of your edit, but in the "Create an alert policy" section, step 2 should there be a space between the icon and New alert policy?
In the "Change an alert policy" section, step 1 I don't think you need parens any more. That was old style guide stuff when we used > for pathing.